src/lib/converter/nodes/module.ts

```diff
@@ -22,20 +22,14 @@ export class ModuleConverter extends ConverterNodeComponent<ts.ModuleDeclaration
      * @return The resulting reflection or NULL.
      */
     convert(context: Context, node: ts.ModuleDeclaration): Reflection {
-        const parent = context.scope;
-        const reflection = createDeclaration(context, node, ReflectionKind.Module);
-
+        const reflection = context.isInherit && context.inheritParent === node
+            ? <DeclarationReflection> context.scope
+            : createDeclaration(context, node, ReflectionKind.Module);
         context.withScope(reflection, () => {
-            if (parent instanceof ProjectReflection && !context.isDeclaration &&
-                (!module || module.valueOf() === ts.ModuleKind.None.valueOf())) {
```

Comment:
`module` referred to the @types/node global. ts.ModuleKind is used only as the compilerOptions type as far as I can see. And valueOf?!